documentation/Using-the-SDK/Creating-a-pipeline.md

```diff
@@ -2,58 +2,62 @@
 
 ## Overview
 
-Data Processing Pipelines (DPP) allow you to compose components together to
-data from inputs into outputs. A DPP is comprised of the following:
-1) Data Source Parser
-2) One or more Data Cookers (Cookers)
+Data Processing Pipelines (DPPs) allow you to compose components together that 
+map data between inputs and outputs. A DPP is comprised of the following:
+1) one __source parser__
+2) one or more `DataCooker`s (cookers)
 
-Cookers can be further broken down into:
-1) Source Cookers
-2) Composite Cookers
+For a real world example of a pipeline, see the LTTng Plugin for WPA, found on GitHub [here](https://github.com/microsoft/Microsoft-Performance-Tools-Linux)
```

Comment:
drop WPA?

Because it works for other things too :)

Reply:
Good catch. Fixed!